cli/src/commands/new/index.ts

```diff
@@ -225,7 +245,7 @@ export default class NewCommand extends BaseCommand {
     }
   }
 
-  private async createApp(name: string, dir: string, sdk: SDK, template: string, force: boolean, prerelease: boolean, createGitRepo: boolean) {
+  private async createApp(name: string, dir: string, sdk: SDK, template: string, _force: boolean, prerelease: boolean, createGitRepo: boolean) {
```

Comment:
I think `force` was left over from a prior refactor.  You can remove it and update the usage.